### DISCUSSION TITLE
Gopro 12 y 13 dormidas

### DISCUSSION THREAD

**Discussion by Ezequiellpi** [gopro/labs#1128] [Q&A]
Good afternoon,
 with the motion detection mode, on both GoPro 12 and 13, when you sleep the screen stops working although it remains on. any idea what's going on? 
greetings and thanks

**Comment by dnewman-gpsw** [comment]
I'm not seeing that here.  In this quick test I let the screen time-out, then move a object in front of the lens and the motion detection is still running correctly.  Using HERO13.

https://github.com/user-attachments/assets/90fbf03a-14ab-4697-aa7d-2308066c7f5e

**Reply by Ezequiellpi** [reply]
correct. The problem comes from leaving it for more than an hour with the screen asleep. Sooner or later, he stops waking up.

**Reply by dnewman-gpsw** [reply]
Interesting.  I will test longer.

**Reply by dnewman-gpsw** [reply]
I just run this script ```"$m motion"!N!R``` measuring the motion.  It run without issue for 2 hours.   I run the old style ```!SM3D4H3!1R``` for an hours with no issues.

**Reply by Riebli** [reply]
Hello Ezequiellpi
Try this link https://[gopro.github.io/labs/control/motion/](https://gopro.github.io/labs/control/motion/)
The app for ios has a problem, it will end up with a !R produces but it should be !1R

**Reply by dnewman-gpsw** [reply]
That is the link I used to generate ```!SM3D4H3!1R```  It work for over an hour.  At what point is it failing?

**Reply by Riebli** [reply]
Hello dnewman-gpsw
If you do the same with the mobile app ios, a different code is generated, see picture.
![IMG_3372](https://github.com/user-attachments/assets/73f6952f-184a-4705-8a0b-a733d8403654)

**Reply by Riebli** [reply]
And it remains standing

**Reply by dnewman-gpsw** [reply]
I know, does !SM3D4H3!1R fail?  Is so when?

**Reply by Riebli** [reply]
Good day
! SM3D4H3!1R This code works fine 
! SM3D4H3!R This code only works at the beginning, after 2.3, times it hangs.
But I saw that it was corrected in the version labs1.20.70.
Thank you very much.
